metaci/api/views/test_testmethod_perf.py

```diff
@@ -0,0 +1,335 @@
+from metaci.conftest import StaffSuperuserFactory, TestResultFactory
```

Comment:
Can you turn metaci/api/tests.py into a package (its existing tests can go in metaci/api/tests/test_urls.py) and move this file into that package? That will better match the structure of tests elsewhere in this project and will make it easier to configure pytest to only pay attention to modules inside folders named `tests`.

Reply:
Done in:

https://github.com/SFDO-Tooling/MetaCI/pull/380/commits/beaf7c37b45bc92fadabf61b899dfb0faaedd09b